### PR TITLE
Fix error messages during sign up

### PR DESCRIPTION
**Pull request by kermit-klein**
[PT LINK](https://www.pivotaltracker.com/story/show/173174931)

Fix sign up error messages

### CODE REVIEW

File: cypress/fixtures/unsuccessful_login.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
-  "success":false, 
-  "errors":["Invalid login credentials. Please try again."]
-}
\ No newline at end of file
+  "success": false,
```
Comment (ViaMarcus) [inline cypress/fixtures/unsuccessful_login.json:2]: Shouldn't this be status: "failure" or something?

File: cypress/integration/userCanRegisterForAccount.feature.js [hunk 2]
```diff
@@ -40,7 +40,7 @@ describe("User can register for an account", () => {
         url: "**/auth",
         response: {
           success: false,
```
Comment (ViaMarcus) [inline cypress/integration/userCanRegisterForAccount.feature.js:42]: Do we get "status": "success" and "success": false on success and fail respectively?

File: cypress/fixtures/unsuccessful_login.json (hunk 1 above)
Comment (kermit-klein) [reply cypress/fixtures/unsuccessful_login.json:2]: Devise sends this response when there is a failed login attempt, that's why we had to change our error handling

File: cypress/integration/userCanRegisterForAccount.feature.js (hunk 2 above)
Comment (kermit-klein) [reply cypress/integration/userCanRegisterForAccount.feature.js:42]: Devise gives the following for registration
status : "error"
status : "success"

and for the login
success:true
success:false



response fixture is incorrect about success key, but it is not checked during error handling, status of the response automatically triggers the catch error, we are not comparing success key anywhere I believe(other than login)

File: src/components/SignUpForm.jsx [hunk 3]
```diff
@@ -40,7 +41,7 @@ const SignUpForm = (props) => {
       <Grid className="signup-container" verticalAlign="middle">
         <Grid.Column align="center">
           {signupMessage !== "" && (
-            <h3 id="error-message">
+            <h3 style={{ color: "black" }} id="error-message">
```
Comment (erikbjoern) [inline src/components/SignUpForm.jsx:44]: We did red text on subscription page, maybe we should do that here as well?

**Review by erikbjoern — APPROVED** [review]

**Review by oliverochman — APPROVED** [review]

**Pull request merged**